periphery/tenderly/dune.js

```diff
@@ -1,36 +1,35 @@
 import axios from 'axios'
 
 const triggerDuneQuery = async (context, event) => {
-  // hash of Bid Event signature: keccak256("Bid(uint256 saleId, address bidder, uint256 amount)");
+  // keccak256(Bid(uint256 saleId,address bidder,uint256 amount));
   const BID_EVENT_SIG =
     '0xdcd726e11f8b5e160f00290f0fe3a1abb547474e53a8e7a8f49a85e7b1ca3199'
 
   const placeBidLog = event.logs.find((log) => log.topics[0] === BID_EVENT_SIG)
-
   if (!placeBidLog) return
+
   const saleId = BigInt(placeBidLog.topics[1]).toString()
-  const apiToken = await context.secrets.get('DUNE_API_KEY')
-  const queryIds = await Promise.all([
-    context.storage.getJson('CROWDSALE_BIDS_QUERY_ID'),
-    context.storage.getJson('CUMULATIVE_BIDS_QUERY_ID')
-  ])
+  const DUNE_API_KEY = await context.secrets.get('DUNE_API_KEY')
+
+  //[Cumulative Bids, CrowdSale Bids]
+  const duneQueryIds = [2709374, 2709364]
```

Comment:
Whats the reason for not getting this out of the context?

Reply:
as said, they're fixed values that aren't supposed to change. I'm not totally aware of the convos we had before but I'd say, if we really were to use a different query id we can also redeploy the tenderly action (we would have to update the tenderly env anyway)

Reply:
>@[elmariachi111](https://github.com/elmariachi111) elmariachi111 [3 days ago](https://github.com/moleculeprotocol/IPNFT/pull/139#discussion_r1266994185)
don't know about dune's capabilities but these seem like context variables to me. not secret but also not necessarily part of the code.

Well depends on the change. But I agree, it's quite unlikely that we will change this w/o making significant updates anyway, which is why I think it's fine to just have them as fixed values in here.